Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -761,6 +763,16 @@ def main():
         email_html = demisto.args().get('emailHTML', "")
         max_urls = int(demisto.args().get('maxNumberOfURL', 5))
         urls_argument = demisto.args().get('urls', '')
+        reliability = demisto.args().get("reliability", DBotScoreReliability.A_PLUS)
+
+        if DBotScoreReliability.is_valid_type(reliability):
+            reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(
+                reliability
```

Comment:
I don't understand the combination of this two functions, why do we need get_dbot_score_reliability_from_str?

Reply:
you are right, this is duplication. I removed one handling of invalid reliability.